authui/src/authflowv2/components/phone-input.css

```diff
@@ -0,0 +1,134 @@
+@layer components {
+  :root {
+    --phone-input__space-x: 0.5em;
+    --phone-input__text-color: var(--color-grey-700);
+    --phone-input__font-size: var(--typography-body-large__font-size);
+    --phone-input__line-height: var(--typography-body-large__line-height);
+    --phone-input__letter-spacing: var(--typography-body-large__letter-spacing);
+    --phone-input__font-weight: var(--typography-body-large__font-weight);
+    --phone-input__font-family: twemoji-countries, Inter;
+
+    --phone-input__trigger__ring-color: var(--color-grey-100);
```

Comment:
We follow BEM and BEM does not have two elements. So it should be named as `--phone-input__trigger-ring-color`.